libraries/helpers.rb

```diff
@@ -58,7 +60,7 @@ def pdns_package_module_requirements
 
 def pdns_source_module_requirements
   modules = node['pdns']['authoritative']['backends']
-  required_packages = []
+  required_packages = %w(autoconf automake bison flex g++ git libboost-all-dev libtool make pkg-config ragel)
```

Comment:
Probably, but I don't think `build-essential` is listed as a requirement anywhere so I went with the minimal set of packages that were needed for this particular build.